test/test_transforms_tensor.py

```diff
@@ -143,132 +143,149 @@ def test_random_autocontrast(self):
+        for transformed_tensor, transformed_tensor_script in zip(transformed_t_list, transformed_t_list_script):
+            assert_equal(
+                transformed_tensor,
+                transformed_tensor_script,
+                msg="{} vs {}".format(transformed_tensor, transformed_tensor_script),
+            )
 
-        tensor, _ = _create_data(h, w, channels=c, device=self.device)
-
-        matrix = torch.rand(c * h * w, c * h * w, device=self.device)
-        mean_vector = torch.rand(c * h * w, device=self.device)
-
-        fn = T.LinearTransformation(matrix, mean_vector)
+        # test for class interface
+        fn = getattr(T, method)(**meth_kwargs)
         scripted_fn = torch.jit.script(fn)
+        output = scripted_fn(tensor)
+        self.assertEqual(len(output), len(transformed_t_list_script))
 
-        _test_transform_vs_scripted(fn, scripted_fn, tensor)
-
-        batch_tensors = torch.rand(4, c, h, w, device=self.device)
-        # We skip some tests from _test_transform_vs_scripted_on_batch as
-        # results for scripted and non-scripted transformations are not exactly the same
-        torch.manual_seed(12)
-        transformed_batch = fn(batch_tensors)
+        # test on batch of tensors
+        batch_tensors = _create_data_batch(height=23, width=34, channels=3, num_samples=4, device=self.device)
         torch.manual_seed(12)
-        s_transformed_batch = scripted_fn(batch_tensors)
-        assert_equal(transformed_batch, s_transformed_batch)
+        transformed_batch_list = fn(batch_tensors)
+
+        for i in range(len(batch_tensors)):
+            img_tensor = batch_tensors[i, ...]
+            torch.manual_seed(12)
+            transformed_img_list = fn(img_tensor)
+            for transformed_img, transformed_batch in zip(transformed_img_list, transformed_batch_list):
+                assert_equal(
+                    transformed_img,
+                    transformed_batch[i, ...],
+                    msg="{} vs {}".format(transformed_img, transformed_batch[i, ...]),
+                )
 
         with get_tmp_dir() as tmp_dir:
-            scripted_fn.save(os.path.join(tmp_dir, "t_norm.pt"))
-
-    def test_compose(self):
-        tensor, _ = _create_data(26, 34, device=self.device)
-        tensor = tensor.to(dtype=torch.float32) / 255.0
-
-        transforms = T.Compose([
-            T.CenterCrop(10),
-            T.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
-        ])
-        s_transforms = torch.nn.Sequential(*transforms.transforms)
+            scripted_fn.save(os.path.join(tmp_dir, "t_op_list_{}.pt".format(method)))
 
-        scripted_fn = torch.jit.script(s_transforms)
-        torch.manual_seed(12)
-        transformed_tensor = transforms(tensor)
-        torch.manual_seed(12)
-        transformed_tensor_script = scripted_fn(tensor)
-        assert_equal(transformed_tensor, transformed_tensor_script, msg="{}".format(transforms))
+    def test_five_crop(self):
```

Comment:
I think this (and some other tests) should be left out, as they've been ported to pytest already